### ISSUE TITLE
[Broken Website] MediaWiki math formulas not inverted

### ISSUE THREAD

**Issue by KaseiFR** [darkreader/darkreader#12018]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this website issue to avoid opening a duplicate.
- [ ] I can reproduce this website issue in a new, unmodified web browser profile with Dark Reader installed as the only extension.

### Website Issue Description

MediaWiki-based wikis can include math formulas. They are not inverted and thus very hard to read on a dark background.

Examples include all the Paradox game wikis, such as this [page](https://vic3.paradoxwikis.com/Market#Market_price) in the Victoria III wiki:
![image](https://github.com/darkreader/darkreader/assets/6408124/870fe4d3-49fb-4f2d-9f4f-5cccddb457fc)

Here it is again with a custom `filter: invert(1)` CSS [rule](https://gist.github.com/KaseiFR/dac3f9cf9ddb39e31c74d030815c15f7):
![image](https://github.com/darkreader/darkreader/assets/6408124/4edd2e79-18fc-46f3-8fed-24f16d97b54b)

I'm not sure what's the best way to apply a rule to every mediawiki site (regardless of domain) using DarkReader, and it might impact other users, hence this issue.

### Website Address

https://vic3.paradoxwikis.com/Market#Market_price

### Steps To Reproduce

Find a math formula

### Operating System

Windows 10

### Web Browser name and version

Firefox 120.0.1

### Dark Reader version

4.9.73

**Comment by Myshor** [comment]
There was fix in #12001 done.

Just check that closed issue and my tip how to have newest site fixes included in Dark Reader.

I just use last sitefixes and for me the formulas work fine.

**Comment by Myshor** [comment]
As there is no complains after my last comment it works fine with latest sitefixes I close issue.

**Comment by Gravifer** [comment]
I would like to report some offending pages:
https://en.wikipedia.org/wiki/Laplace_operator, in which *some* formulae are correctly inverted, but others not, which I find baffling;
https://en.wikipedia.org/wiki/Fredholm_integral_equation, in which all formulae remain dark on a dark background.
I indeed employed the remedy suggested by @Myshor, but in vain. This can be reproduced across multiple devices I have, and resetting all fixes in the developer tool doesn't make any changes either.

I used to manually make fixes to cope with things like this, but it seems that the formulae entities on MediaWiki sites have become more complicated, and I cannot figure out this one. Any help is appreciated.
